src/commands/metafrage.ts

```diff
@@ -0,0 +1,79 @@
+"use strict";
+
+// ========================= //
+// = Copyright (c) NullDev = //
+// ========================= //
+
+// Dependencies
+let parseOptions = require("minimist");
+
+/**
+ * Sends instructions on how to ask better questions
+ * @param {import("discord.js").Client} client
+ * @param {import("discord.js").Message} message
+ * @param {Array} args
+ * @param {Function} callback
+ * @returns {Function} callback
+ */
+exports.run = (client: any, message: any, args: any, callback: any) => {
+    // parse options
+    let options = parseOptions(args, {
+        "boolean": [
+            "english"
+        ],
+        alias: {
+            english: "e"
+        }
+    });
+
+    if (args.length === 0 || (args.length === 1 && options.english)){
+        // insult collections, feel free to expand
+        let germanInsults: string[] = [
+            "du Sohn einer ranzigen Hafendirne!",
+            "möge dich der Blitz beim scheißen treffen!",
+            "du verdammter Troglodyt!",
+            "sonst muss ich heute Nacht noch deine Mama besuchen!"
+        ];
+        let englishInsults: string[] = [
+            "you fucking imbecile!",
+            "retard!",
+            "you troglodyte!"
+        ];
+        // by default, just pick the first or a fixed insult
+        let choice: number = 0;
+        let insult: string = "Idiot!";
+
+        if (!options.english){
+            // insult user, then explain meta questions
+            if (germanInsults.length !== undefined){
+                choice = Math.max(choice, Math.floor(Math.random() * germanInsults.length));
+                insult = germanInsults[choice];
+            }
+            message.channel.send(`
+            Hör auf, Metafragen zu stellen, ${insult}
+            Das ist reine Zeitverschwendung und hindert uns nur daran, ~~uns zu beleidigen~~ an echten Problemen zu arbeiten.
+            Für Tipps zum besser machen: <https://metafrage.de>
+            `);
+        }
+        else{
+            // insult user, then explain meta questions - but this time in english!
+            if (englishInsults.length !== undefined){
+                choice = Math.max(choice, Math.floor(Math.random() * englishInsults.length));
+                insult = englishInsults[choice];
+            }
+            message.channel.send(`
+            Stop asking meta questions, ${insult}
```

Comment:
Durch deinen check in l. 60 kann insult hier undefined sein, wenn man annimmt, dass dein Array leer ist. Wäre ein default Wert nicht angebracht? Oder du lässt den Check einfach direkt weg, weil wir ja wissen, dass das Array nicht leer ist.

Reply:
Defaultwert wird in L44 initialisiert, der Check soll ja nur entscheiden, ob Default oder aus der Liste beleidigt wird.